python/foglamp/services/south/server.py

```diff
@@ -110,6 +111,12 @@ def __init__(self):
 
             # Plugin initialization
             self._plugin_info = self._plugin.plugin_info()
+            if self._plugin_info['version'].startswith('2') and self._plugin_info['mode'] == 'async':
```

Comment:
May be to keep version things in sync we shall need to check `>= 1.5` i.e. float value comparison of first 3 characters in version